helix-view/src/editor.rs

```diff
@@ -34,6 +34,42 @@ where
     Ok(Duration::from_millis(millis))
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+#[serde(rename_all = "kebab-case", default, deny_unknown_fields)]
+pub struct FilePickerConfig {
+    pub hidden: bool,
+    pub parents: bool,
+    pub ignore: bool,
+    pub git_ignore: bool,
+    pub git_global: bool,
+    pub git_exclude: bool,
+    pub max_depth: Option<usize>,
+}
+
+impl Default for FilePickerConfig {
+    fn default() -> Self {
+        Self {
+            // IgnoreOptions
+            // Enables ignoring hidden files.
+            hidden: true,
+            // Enables reading ignore files from parent directories.
+            parents: true,
+            // Enables reading `.ignore` files.
+            ignore: true,
+            // Enables reading `.gitignore` files.
+            /// Whether to hide files in .gitignore from displaying in file picker. Defaults to false.
+            git_ignore: true,
+            // Enables reading global .gitignore, whose path is specified in git's config: `core.excludefile` option.
+            git_global: true,
+            // Enables reading `.git/info/exclude` files.
+            git_exclude: true,
+            // WalkBuilder options
+            // Maximum Depth to recurse.
+            max_depth: None,
```

Comment:
These comments should really be doc comments (`///`) on the `FilePickerConfig` struct, that way they show in crate docs.

Reply:
OK, I saw that was the way with previous config, now that makes sense. I'll make an adjustment.